Assets/Scripts/EnemyController.cs

```diff
@@ -0,0 +1,54 @@
+using System.Collections;
+using System.Collections.Generic;
+using Unity.VisualScripting;
+using UnityEngine;
+
+public class EnemyController : MonoBehaviour
```

Comment:
Currently this class wouldn't be usable (empty AttackPlayer method and no movement) would the intention be to add these later to this class, or just inherit this and implement it in the child class?

If inherit this should be an abstract class